tests/playwright/src/special-specs/installation/podman-installer.spec.ts

```
@@ -75,9 +76,17 @@ test.describe
       console.log(
         `Trying to find podman installer artifact: ${podmanInstallerFilePrefix}-${archPart}.${fileFormatRegexp}`,
       );
-      const electronBinary = await runner.getElectronApp().evaluate(async ({ app }) => {
-        return app.getPath('exe');
-      });
+      let electronBinary;
```

Biome (2.4.9)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)